docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -1,8 +1,8 @@
 [[example-using-index-lifecycle-policy]]
-=== Tutorial: Customize built-in {ilm-init} policies
+=== Tutorial: Customize agent data stream {ilm-init} policies
 
 ++++
-<titleabbrev>Tutorial: Customize built-in policies</titleabbrev>
+<titleabbrev>Tutorial: Customize agent data stream ilm policies</titleabbrev>
```

Comment:
```suggestion
<titleabbrev>Tutorial: Customize agent data stream ILM policies</titleabbrev>
```

Reply:
I think `{ilm-init}` is what we'd normally use here, right?

Reply:
I'm not certain if this prop works inside of the title abbreviation tag (which appears in the sidebar). It would be worth the time to use the variable and spot check it though 👍 

```suggestion
<titleabbrev>Tutorial: Customize agent data stream {ilm-init} policies</titleabbrev>
```